### ISSUE TITLE
key binding collision

### ISSUE THREAD

**Issue by Chrisriis** [wbond/sublime_alignment#80]
The default Mac key-binding which is ctrl+cmd+a collides with Sublime Linter. 
I therefore added my own keybinding like this which works, but it should be fixed though:

[
    { "keys": ["super+ctrl+b"], "command": "alignment" },
]

**Comment by wbond** [comment]
I don't think "fixed" is the right word – the alignment key binding has been that way since September 2011, since before SublimeLinter existed.

I'm not sure there is a good solution for this other than perhaps a message to users warning them that SublimeLinter uses the same binding.

**Comment by Chrisriis** [comment]
I totally agree. I will make the SublimeLinter guys aware of this issue as well :+1:

**Comment by groteworld** [comment]
Thanks for bringing this to our attention, @Chrisriis. I will start working towards finding a suitable replacement as we (SL) would rather not overlap other common packages.

**Comment by TMMC** [comment]
Additionally in polish right alt + letter 'a' are used to get character 'ą', so maybe default keymapping could get the difference between ralt and lalt? Or maybe could defaults be changed? I used Package Resource Viewer to get rid of it locally.

**Comment by wbond** [comment]
@TMMC I think your comment is a duplicate of #39

**Comment by TMMC** [comment]
Yes, sorry for that, I must have overlooked it, so I'm removing it.

**Comment by christianesperar** [comment]
+1

**Comment by romellem** [comment]
For anyone who likes accessing commands directly with **⌘ + SHIFT + P**, but was annoyed to see that **Alignment: Align** was not available, I found a work-around.

1. Install **Edit Command Palette** from Package manager ([source](https://github.com/twolfson/sublime-edit-command-palette)).
2. Open Command Palette with **⌘ + SHIFT + P**, and type `Commands` and select "**Preferences: Commands - User**"
3. Add the following command to your **Default.sublime-commands** file<br>
  ```
{
    "caption": "Alignment: Align",
    "command": "alignment"
}
  ```
4. Now, when opening Command Palette, you can align your selection by typing "**Align**" and selecting it from the search results.
